bedshift/bedshift.py

```diff
@@ -77,6 +77,9 @@ def build_argparser():
     parser.add_argument(
         "--addfile", type=str, help="Add regions from a bedfile")
 
+    parser.add_argument(
+        "--valid_regions", type=str, help="valid regions in which regions can be randomly added")
```

Comment:
Most frequently, CLI arguments are with hyphens rather than underscores, so it would be `--valid-regions`.